crates/aptos/src/node/mod.rs

```diff
@@ -37,11 +37,7 @@ use tokio::time::Instant;
 ///
 #[derive(Parser)]
 pub enum NodeTool {
-    AddStake(AddStake),
-    UnlockStake(UnlockStake),
-    WithdrawStake(WithdrawStake),
-    IncreaseLockup(IncreaseLockup),
-    RegisterValidatorCandidate(RegisterValidatorCandidate),
+    InitializeValidator(InitializeValidator),
```

Comment:
Can you add RotateConsensusKey and update_network_and_fullnode_addresses as well?

Reply:
I have those in this PR below,  but in node subcommand.
I've separated things so that stake subcommand is for owner of the stake,  and node subcommand is for the validator operator. 

Check out the unit test,  it calls all those.

Reply:
UpdateValidatorNetworkAddresses is already there
UpdateConsensusKey was added below, but forgot to add it here, tnx